services/tests/integration/gateway/policy-directive.spec.ts

```diff
@@ -219,6 +222,7 @@ const testCases: [string, TestCase][] = [
   [
     'Stub, policy, lowerCase on field (ALLOW)',
     {
+      // This is incorrect order of directives but it still does work by chance.
```

Comment:
I don't think `by chance` is the correct term here, it still works because the `lowerCase` directive specifically calls the next resolver, but might not work with another directive after it